pkg/controller/mcmhub/ansiblejob.go

```diff
@@ -78,14 +79,14 @@ func (jIns *JobInstances) registryJobs(gClt GitOps, subIns *subv1.Subscription,
 		jobRecords.mux.Lock()
 		jobRecords.Original = ins
 
-		if forceRegister && len(jobRecords.Instance) != 0 {
+		if placementDecisionUpdated && len(jobRecords.Instance) != 0 {
 			plrSuffixFunc := func() string {
 				return fmt.Sprintf("-%v-%v", subIns.GetGeneration(), placementRuleRv)
 			}
 
 			suffix = plrSuffixFunc()
 
-			logger.Info("forceRegister suffix is: " + suffix)
+			logger.V(3).Info("placementDecisionUpdated suffix is: " + suffix)
```

Comment:
I dont't think `V(3)` will work for us, if you want it to be noise then `V(InfoLog)`, less noise `V(DebugLog)`

Reply:
Using `DebugLog` now.